### ISSUE TITLE
Extraneous dependency on pytzdata still exists

### ISSUE THREAD

**Issue by joaoe** [sdispater/pendulum#657]
- [x] I am on the [latest](https://github.com/sdispater/pendulum/releases/latest) Pendulum version.
- [x] I have searched the [issues](https://github.com/sdispater/pendulum/issues) of this repo and believe that this is not a duplicate.

- **OS version and name**: any (I'm using Windows)
- **Pendulum version**: 2.1.2
## Issue

Hi.

[This commit](https://github.com/sdispater/pendulum/commit/ae2c9f59beface872ec54c30b943a0bc3ad77273) from mid summer removed the dependency on `pytzdata` and replaced it with `tzdata`. The former is a 3rd party python module. The latter is the official timezone database mentioned in the [Python docs](https://docs.python.org/3.9/whatsnew/3.9.html#zoneinfo)

However,
1. The [wheel files at pypi.org](https://pypi.org/project/pendulum/#files) still list pytzdata as the dependency . For instance, download [pendulum-2.1.2.tar.gz](https://files.pythonhosted.org/packages/db/15/6e89ae7cde7907118769ed3d2481566d05b5fd362724025198bb95faf599/pendulum-2.1.2.tar.gz), open the PKG-INFO file inside, and see this
```
Requires-Dist: python-dateutil (>=2.6,<3.0)
Requires-Dist: pytzdata (>=2020.1)
Requires-Dist: typing (>=3.6,<4.0); python_version < "3.5"
```
That is incorrect.

2. I tried to find references to `pytzdata` in the repo and there is a single mention of `pytzdata` in the `[tools.isort]` section https://github.com/sdispater/pendulum/blob/42865d635f9bdf9cda61de9fc4d25e0568274867/pyproject.toml#L89 That should be removed.

**Comment by joaoe** [comment]
Oh, I just noticed that the 2.1.2 version on pypi.org is two years old and not from July 2022 ! Is this being maintained ?

**Comment by grubberr** [comment]
Hello, we already started to experience some problem with it

```
docker run -ti --rm python:3.11-slim /bin/bash
pip3 install pendulum
TZ="Europe/Kyiv" python3 -c "import pendulum; pendulum.now()"

    ...
    raise InvalidTimezone(timezone)
pendulum.tz.zoneinfo.exceptions.InvalidTimezone: Invalid timezone "Europe/Kyiv"
```

**Comment by Secrus** [comment]
We are currently working on version 3.0, which takes some time, especially since both sdispater and I are also Poetry maintainers, so our time is stretched thin as it is.

**Comment by NickFabry** [comment]
As a temporary fix, to force pendulum 2.1.2 to use either the system zoneinfo, or (on Windows), the zoneinfo from the first party module tzdata, put the following into your python import path as (for example) pendulum_212_fix.py, and after you `import pendulum`, add the line `import pendulum_212_fix`, and you should be good.

```
#!/usr/bin/env python
# -*- coding: utf-8 -*-

"""
This module has one purpose - to fix the behavior of pendulum release
2.1.2, which uses a hardcoded version of the IANA zoneinfo data, 2022.2.
Importing this module after importing pendulum should cause pendulum to
use either the OS provided zoneinfo data, or if that does not exist, to
use the zoneinfo from the first party module tzdata. Note this introduces
a dependence on tzdata, which must be separetely installed, typically via
pip install tzdata.

In theory, IMMEDIATELY AFTER IMPORTING pendulum, you should simply
import this module and pendulum will then use the local system
zoneinfo, if available, and if not, the zoneinfo from tzdata, when
creating aware datetimes.
"""



import pathlib
import sysconfig
import tzdata

LOCAL_TZPATH = sysconfig.get_config_var('TZPATH')
if LOCAL_TZPATH:
    # This could be a colon separated list of paths; we take the first only
    LOCAL_TZPATH = LOCAL_TZPATH.split(':')[0]
else:
    LOCAL_TZPATH = pathlib.Path(tzdata.__path__) / 'zoneinfo'

import pendulum
import pytzdata
pytzdata.set_directory(LOCAL_TZPATH)



if __name__ == '__main__':
    pass
```

**Comment by Secrus** [comment]
Fixed in Pendulum 3.0
